website/src/assets/custom-theme.ts

```diff
@@ -0,0 +1,145 @@
+import { extendTheme } from '@chakra-ui/vue-next'
+import { mode } from '@chakra-ui/vue-theme-tools'
+
+export default extendTheme({
+  colors: {
+    vue: {
+      50: '#EEF8F4',
+      100: '#CDEBDD',
+      200: '#ACDEC6',
+      300: '#8BD1AF',
+      400: '#6AC398',
+      500: '#42B883',
+      600: '#30855F',
+      700: '#2E7452',
+      800: '#21533B',
+      900: '#143223',
+    },
+  },
+  fonts: {
+    heading: 'Inter, sans-serif',
+    body: 'Inter, sans-serif',
+  },
```

Comment:
I think we should have a fallback font in case the font is not yet loaded in some devices with slow connections. You can see them in the default theme